src/frontend/src/utils/iiConnection.ts

```diff
@@ -358,11 +341,39 @@ export class IIConnection {
     }
     const actor = Actor.createActor<_SERVICE>(internet_identity_idl, {
       agent,
-      canisterId: canisterId,
+      canisterId: this.canisterId,
     });
     return actor;
-  }
+  };
 
+  requestFEDelegation = async (
```

Comment:
this isn't new code; just moved to the class before it needs to read the canisterId